Formula/goad.rb

```diff
@@ -0,0 +1,24 @@
+class Goad < Formula
+  desc "AWS Lambda powered, highly distributed, load testing tool built in Go."
+  homepage "https://goad.io/"
+  url "https://github.com/goadapp/goad.git"
+  version "1.4.0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = buildpath/"bin"
+    ENV.prepend_create_path "PATH", buildpath/"bin"
```

Comment:
Right now this line is not obsolete because we rely on `go-bindata` in the build process.